frontend/src/pages/preferences.vue

```diff
@@ -2,25 +2,65 @@
   <VContentPage>
     <h1>{{ $t("pref-page.title") }}</h1>
 
-    <div v-for="isSwitchable in [false, true]" :key="isSwitchable">
-      <h2>
+    <!-- TODO: Extract this to preferences modal. -->
```

Comment:
@zackkrida only this bit is what will be made public later on.

Reply:
Definitely! I understand now. I incorrectly assumed this PR was suggesting the page be used publicly. We can save my suggestions for if (and when) that day comes. Clever to implement some of the early design work for preferences in this PR.